### PR TITLE
fix(typography): bold style using b or strong tag (VIV-1317)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/styles/src/lib/core/typography.scss [hunk 1]
```diff
@@ -76,68 +76,63 @@ $selectors: (
...
+	h2,
+	.heading2,
+	h3,
+	.heading3,
+	h4,
+	.heading4 {
+
+		#{map.get($selectors, "bold")} {
+			font-weight: 500;
+		}
```
Comment (RichardHelm) [inline libs/styles/src/lib/core/typography.scss:123]: Why add this? Headings have weight 500, so then the bold tags are not going to do anything inside headings

File: libs/styles/src/lib/core/typography.scss [hunk 2]
```diff
@@ -66,7 +66,7 @@ $selectors: (
 	}
 
 	#{map.get($selectors, "bold")} {
-		font: var(#{constants.$vvd-typography-base-bold});
+		font-weight: 600;
```
Comment (RichardHelm) [inline libs/styles/src/lib/core/typography.scss:69]: Let's leave this for now, but this should probably be a token

**Review by RichardHelm — CHANGES_REQUESTED** [review]

File: libs/styles/src/lib/core/typography.scss (hunk 1 above)
Comment (rachelbt) [reply libs/styles/src/lib/core/typography.scss:123]: but they do!
If this is bot here - it gets 600 - and its extra bold in the headers

File: libs/styles/src/lib/core/typography.scss (hunk 2 above)
Comment (RichardHelm) [reply libs/styles/src/lib/core/typography.scss:69]: Yes I understand. I mean the value 600 should be from a token as it should be configurable.

So if someone wants their bold text to be 700 they can change `typography-base-bold` but here it remains 600

File: libs/styles/src/lib/core/typography.scss (hunk 1 above)
Comment (RichardHelm) [reply libs/styles/src/lib/core/typography.scss:123]: Yes it becomes extra bold as expected when using bold tags.

But I can see the argument headings are already considered bold, so bold tags shouldn't do anything so I will approve this

**Review by RichardHelm — APPROVED** [review]

File: libs/styles/src/lib/core/typography.scss (hunk 2 above)
Comment (rachelbt) [reply libs/styles/src/lib/core/typography.scss:69]: bold is 600, 700 is extra-bold.
Either way - if someone is using <b> and want it to be 700, then he or she can run this over with css.
I'm not sure how common is the use of <b> or <strong> anyway.

**Pull request merged**